src/views/category_panel.dot

```diff
@@ -49,7 +49,7 @@
               {{?}}
 
               {{? grades }}
-                <div class="panel__note">
+                <div class="panel__note" {{= click(this.goToFeedbackUrl, this, poi) }}>
```

Comment:
Wouldn't it be semantically more correct to use an `a` tag here and build the `href` dynamically ?

Reply:
There are some logic instructions behind the `feedbackUrl` that needs to be scoped into a function in my opinion, instead of splitting strings directly in the template 😬